java/api/src/main/java/io/ray/api/options/CallOptions.java

```diff
@@ -1,23 +1,31 @@
 package io.ray.api.options;
 
+import io.ray.api.placementgroup.PlacementGroup;
 import java.util.HashMap;
 import java.util.Map;
 
 /** The options for RayCall. */
 public class CallOptions extends BaseTaskOptions {
 
   public final String name;
+  public final PlacementGroup group;
+  public final int bundleIndex;
 
-  private CallOptions(String name, Map<String, Double> resources) {
+  private CallOptions(
+      String name, Map<String, Double> resources, PlacementGroup group, int bundleIndex) {
     super(resources);
     this.name = name;
+    this.group = group;
+    this.bundleIndex = bundleIndex;
```

Comment:
`CallOptions` is for both normal tasks and actor tasks. Since setting a placement group on an actor task is not allowed, I think we need to split `CallOptions` into two classes: `TaskCallOptions` and `ActorTaskCallOptions`. cc @chaokunyang about potential Java API change.

Reply:
Yeah, that's a very good point, thanks for your reminder~

Reply:
Non-exist problem after discussed offline.